iree/compiler/Translation/CMakeLists.txt

```diff
@@ -14,6 +14,12 @@
 
 iree_add_all_subdirs()
 
+if(IREE_ENABLE_EMITC)
+  set(IREE_VM_CONDITIONAL_TARGETS
+    iree::compiler::Dialect::VM::Target::C
+  )
+endif()
```

Comment:
add `bazel_to_cmake: DO NOT EDIT` (or `.skip_bazel_to_cmake` file)

Reply:
:-/ This is a fair amount of skipping. Would be nice to get conditional libraries translated somehow

Reply:
Indeed, I would prefer conditional libraries as well :-/

Reply:
File an issue with some context? There are a few pieces missing from conversion that would help out across the project, someone could work on them in a batch. (`IREE_HAL_DRIVERS_TO_BUILD ` handling also comes to mind)